rust/noosphere-gateway/src/worker/name_system.rs

```diff
@@ -292,16 +307,12 @@ where
         let next_record =
             match fetch_record(client.clone(), name.clone(), identity.did.clone()).await? {
                 Some(record) => {
```

Comment:
This file is noisy due to restructuring processing: adding validation check here